lib/sirsi_holding.rb

```diff
@@ -24,7 +24,7 @@ class SirsiHolding
   attr_reader :id, :current_location, :home_location, :library, :scheme, :type, :barcode, :public_note, :tag, :course_reserves
 
   # rubocop:disable Metrics/ParameterLists
-  def initialize(tag:, call_number:, home_location:, library:, barcode:, scheme: nil, current_location: nil,
+  def initialize(call_number:, home_location:, library:, barcode:, tag: nil, scheme: nil, current_location: nil,
```

Comment:
Why are we allowing tag to be nil now?  There is no place in the code where we don't provide this.

Reply:
I don't understand what `tag` is supposed to be used for, the test in #1057 didn't provide it, and we're not doing anything with it. That makes it seem pretty optional to me?